build.gradle

```diff
@@ -25,9 +24,6 @@ repositories {
         name 'LDTTeam - Modding'
         url 'https://ldtteam.jfrog.io/ldtteam/modding/'
     }
-    flatDir {
```

Comment:
Please don't remove libs, I use them quite often

Reply:
You can use libs via mavenLocal.
That also allows you to use fg.deobf.

Never use a flatDir since FG3 does not support that officially and also not fully

Reply:
Welp, I keep using flatDir since 1.13 without single issue
I don't have set up mavenLocal but guess I should learn something new...